Makefile

```diff
@@ -307,6 +326,30 @@ karma: build
 .PHONY: build
 build: add-on standalone ui vendor_libs
 
+GIT_EXPORT_LOCATION := ../gecko-dev
+GIT_EXPORT_DIR := $(GIT_EXPORT_LOCATION)/browser/extensions/loop
+
+.PHONY: git-export
```

Comment:
Exporting to a git based mozilla-central, although I haven't hooked up any git commands yet, so we could possibly rename it.